mautic/mautic
# Dynamic content bugfix and new features

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | y
| Related user documentation PR URL | n
| Related developer documentation PR URL | n
| Issues addressed (#s or URLs) | n
| BC breaks? | n
| Deprecations? | n

PR:

- bugfix: 2 and more different dynamic contents on tracking page
- allow add push dynamic content action everywhere in campaign (without depends on decision)
- add new action - remove dynamic content
- add schedule trigger mode for both actions 

#### Steps to test bug:
1.  Create Campaign with 2 different dynamic contents (decision + action) 
2.  After PR works as we expect.

Steps to test new features:

Create campagin and test Push dynamic content and Remove dynamic content

![remove-dy](https://cloud.githubusercontent.com/assets/462477/23346005/7e5ae2dc-fc96-11e6-84ea-2b50fb2399d3.PNG)

![dynamic](https://cloud.githubusercontent.com/assets/462477/23218650/0da5bfb6-f91d-11e6-84a7-3b7bcb32e1b5.PNG)
